parsec/core/fs/userfs/userfs.py

```diff
@@ -220,12 +221,17 @@ def user_manifest_id(self) -> EntryID:
 
     def get_user_manifest(self) -> LocalUserManifest:
         """
-        Raises: Nothing !
+        Raises: ValueError
         """
+        if self.storage is None:
```

Comment:
This shouldn't be needed given we are supposed to always instantiate `UserFS` by doing:
```python
async with UserFS.run() as userfs:
    ...
```
And `UserFS.run` setup storage field.

We could add a comment in UserFS.__init__ about this, or just setup the typing of this variable:
```python
class UserFS:
    def __init__(self, ...):
        [...]
        self.storage: UserStorage  # Setup by UserStorage.run factory
        [...]
```

Reply:
done